tinygrad/codegen/uops.py

```diff
@@ -287,6 +288,11 @@ def uop_alu_idx(a: UOp, b, op, dtype=dtypes.int32):
         final_op = self.add(UOps.ALU, where.dtype, (maybe_cast, where.vin[1]), BinaryOps.MUL)
         self.uops = self.uops + after_split_ops
         self.replace_op(where, final_op)
+        for op in self.get_recursive_children(final_op):
+          if (op.arg is BinaryOps.ADD and any([u.uop is UOps.PHI for u in self.get_recursive_children(op)]) and op not in adjusted_adds
```

Comment:
this feels very specific, is it the most generic form for the case?

Reply:
here I'm attempting to get any add op between the WHERE and corresponding PHI, I haven't figured out a beautiful way to do it